input.go

```diff
@@ -155,27 +161,34 @@ func (i *Input) Prompt(config *PromptConfig) (interface{}, error) {
 		defer cursor.Show() // show the cursor when we're done
 	}
 
-	// start waiting for input
+	var line []rune
+
 	for {
-		r, _, err := rr.ReadRune()
-		if err != nil {
-			return "", err
+		if i.options != nil {
+			line = []rune{}
 		}
-		if r == terminal.KeyInterrupt {
-			return "", terminal.InterruptErr
-		}
-		if r == terminal.KeyEndTransmission {
-			break
+
+		line, err = rr.ReadLineWithDefault(0, line, i.onRune(config))
+		if err == readLineAgain {
+			continue
 		}
 
-		b, err := i.OnChange(r, config)
 		if err != nil {
 			return "", err
 		}
 
-		if b {
-			break
-		}
+		break
+	}
+
+	i.answer = string(line)
+	// readline print an empty line, go up before we render the follow up
+	cursor.Up(1)
+
+	// if we ran into the help string
+	if i.answer == config.HelpInput && i.Help != "" {
```

Comment:
@AlecAivazis you mean, if the person send `?` as the response two times in sequence, than we would accept it?

it is not a complex thing to do, i am just thinking that it was a "auto-validation" of the input before... if we change it now, it would break some expectations right?

Reply:
That was my thought but i think it's probably better to leave it for another time. No one has brought it up before so it's probably not worth breaking anyone's tests if they are assuming the current behavior.